### DISCUSSION TITLE
Ability to annotate images

### DISCUSSION THREAD

**Discussion by 3goats** [squidfunk/mkdocs-material#5995] [General]
Hi, any plans to add annotations to images such as PNG's, SVG's etc ? This feature would be awesome.

**Comment by squidfunk** [comment]
Thanks for asking. No, currently no plans, as it's unclear how this should work from an authoring perspective. It might be possible to pull this off with the help of a third party plugin or Markdown extension, but we're currently not considering adding this functionality.
